Software/Python/package_modules.txt

```diff
@@ -0,0 +1,26 @@
+Adafruit_I2C
```

Comment:
I think Adafruit_I2C is now in DI_Sensors.

Reply:
It looks like there's no `Adafruit_I2C` module in here:
https://github.com/DexterInd/DI_Sensors/tree/master/Python/di_sensors/DHT_Sensor/Adafruit_DHT

Hopefully I haven't missed anything when looking over DI_Sensors. What do you think?

Reply:
I know we moved them out! where?

Reply:
it gets installed in Dexter/lib/Adafruit

Reply:
Could it be script_tools?